packages/renderer/src/lib/troubleshooting/TroubleshootingRepairCleanup.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+import { faBroom, faWarning } from '@fortawesome/free-solid-svg-icons';
+import type { ProviderInfo } from '../../../../main/src/plugin/api/provider-info';
+import Button from '../ui/Button.svelte';
+import Fa from 'svelte-fa';
+
+export let providers: ProviderInfo[] = [];
+let providerIdsWithCleanup: string[] = [];
+$: providerIdsWithCleanup = providers.filter(provider => provider.cleanupSupport).map(provider => provider.internalId);
+
+let cleanupInProgress = false;
+
+let cleanupFailures: string[] = [];
+
+async function openCleanupDialog(): Promise<void> {
+  let message = 'This action may delete data. Proceed ?';
+
+  const result = await window.showMessageBox({
+    title: 'Cleanup',
+    message: message,
+    buttons: ['Yes', 'No'],
+  });
+
+  if (result?.response === 0) {
+    cleanup();
+  }
+}
+
+async function cleanup() {
+  try {
+    cleanupInProgress = true;
+    cleanupFailures = [];
+    await window.cleanupProviders(providerIdsWithCleanup, Symbol('cleanup-provider'), (_key, _eventName, args) => {
+      // log into the console as no UI is available to display logs
+      console.log('cleanup event', args);
+    });
+  } catch (e) {
+    console.error(e);
+    cleanupFailures.push(String(e));
+  } finally {
+    cleanupInProgress = false;
+  }
+}
+</script>
+
+<div class="flex flex-row items-center">
+  <div>
+    <div class="text-gray-700 flex flex-row items-center">Clean / Purge data</div>
```

Comment:
Since the button and dialog are called 'cleanup', maybe this should be too? i.e. "Cleanup / Purge data"

Reply:
I think it was the first title but then moved to a shorter one
so going back to `Cleanup / Purge data`